src/include/mpir_misc.h

```diff
@@ -91,11 +91,6 @@ int MPIR_Get_intranode_rank(MPIR_Comm * comm_ptr, int r);
 int MPIR_Close_port_impl(const char *port_name);
 int MPIR_Open_port_impl(MPIR_Info * info_ptr, char *port_name);
 
-int MPIR_Comm_join(int fd, MPI_Comm * intercomm);
-int MPIR_Comm_disconnect(MPIR_Comm * comm_ptr);
-int MPIR_Publish_name(const char *service_name, MPIR_Info * info_ptr, const char *port_name);
-int MPIR_Unpublish_name(const char *service_name, MPIR_Info * info_ptr, const char *port_name);
-
 /* Default routines for asynchronous progress thread */
```

Comment:
Why do we remove these MPIR_ functions?

Reply:
They are normalized to e.g. `MPIR_Comm_join_impl` and prototypes are automatically generated by the python scripts in `src/include/mpir_impl.h`.